src/libs/Together/openapi.yaml

```
@@ -1511,6 +1511,55 @@ paths:
+      parameters:
+        - name: id
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: force
+          in: query
+          required: true
+          schema:
+            type: boolean
+            default: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)